.github/workflows/build.yml

```diff
@@ -24,8 +24,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-        with:
-          submodules: true
+
+      - name: Free space
+        run: |
+          sudo rm -rf /usr/local/lib/android/sdk /opt/hostedtoolcache /usr/share/swift /opt/az
+          sudo apt-get purge -y gcc
```

Comment:
non-related to Free space step?

Reply:
It's related, I remove gcc for conda build. As for the "rm" line, I will try to keep this step the same for all containers (and this removal is needed for docker containers).

Reply:
Now if someone again will add the call to OS gcc (not conda gcc), then the build will report that

Reply:
Should I also include this into PR https://github.com/intel-ai/hdk/commit/25adfaa10f466c3a94b1412d4de92f1e983a86f1 ?